main/main.cpp

```diff
@@ -1859,18 +1860,25 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 			driver_hints_arr.push_back("d3d12");
 #endif
 			driver_hints_with_d3d12 = String(",").join(driver_hints_arr);
+
+#ifdef METAL_ENABLED
+			// make metal the preferred and default driver
+			driver_hints_arr.insert(0, "metal");
+#endif
```

Comment:
I don't know if this is ok, so I wanted to call it out

Reply:
It makes sense!